### ISSUE TITLE
common-kotlin:DuplicatedBlocks should ignore imports

### ISSUE THREAD

**Issue by andreicristianpetcu** [arturbosch/sonar-kotlin#76]
Hi all,

We have a bunch of imports in various Koltin files that are marked as duplicate by the "common-kotlin:DuplicatedBlocks" rule. I think imports should be ignored by this rule, unless you have a better way to avoid this.

Please make it so that the common-kotlin:DuplicatedBlocks rule ignores the imports.

Thank you

**Comment by andreicristianpetcu** [comment]
```kotlin
package com.mycompany;

import org.apache.commons.io.FileUtils
import org.w3c.dom.Document
import java.io.File
import java.io.StringWriter
import javax.inject.Inject
import javax.inject.Singleton
import javax.xml.transform.TransformerFactory
import javax.xml.transform.dom.DOMSource
import javax.xml.transform.stream.StreamResult
import javax.xml.transform.stream.StreamSource
```

Just add the above code in several files.

**Comment by andreicristianpetcu** [comment]
Sorry for the wrong report location. I reported it here now. https://community.sonarsource.com/t/common-kotlin-duplicatedblocks-should-ignore-imports/2539
